pkg/seeds/seeds.go

```diff
@@ -199,7 +199,8 @@ func createVersionArray(existingVersionArray *[]string) []string {
 		return *existingVersionArray
 	}
 	versionArray := make([]string, 0)
-	for k := rand.Intn(4); k < len(config.DistributionVersions); k++ {
+	// The below + 1 ensures that the "any" option is ignored here
```

Comment:
would it be useful to be able to seed with "any"?

Reply:
so in cases of array length 0, then you would receive an array with ["any"]

Reply:
will check this.

Reply:
Right now, the default value I see being set here is `{}`.

The "default" field tag gorm uses will replace any field with that default, if that field is a "zero" value. For a slice, that would be `nil`. However, even when I replace the output of this function with `nil`, the value I get in the database it still `{}`.

I'm not sure what's going on there, but anyway the function isn't working quite like you expect.

Reply:
I should add, you wouldn't see this issue in the handler layer, because the handler calls `FillDefaults()`, which sets the default value before gorm sees it.